packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -1681,7 +1683,9 @@ export default class BackupNg {
                     this._deleteVms(xapi, oldVms)
                   )
 
-                const deleteFirst = getSetting(settings, 'deleteFirst', [srId])
+                const deleteFirst = getSetting(settings, 'deleteFirst', [
+                  srUuid,
```

Comment:
It's manipulable by `XO` and the `SR` list contains IDs. Do see it have sense in this case to use an ID?

Reply:
I agree :+1: